### ISSUE TITLE
No code inspections for build-time errors

### ISSUE THREAD

**Issue by tkapustina** [SpecFlowOSS/SpecFlow.Rider#98]
STR:
1. .feature
```
Background:        
Scenario:         
Given a blog post named "Random" with Markdown body
```
2. Build

ACTUAL:
There is nothing highlighted in the editor.
> SpecFlowFeature1.feature(59, 5): (59:5): expected: #EOF, #TableRow, #TagLine, #ExamplesLine, #ScenarioLine, #RuleLine, #Comment, #Empty, got 'Background:'

EXPECTED:
It would be nice to have code inspections for that.

**Comment by theazort** [comment]
Change from nice to have. This is a recommended practice by JetBrains.

**Comment by tkapustina** [comment]
Example of missing inspections: 
- Scenario without title
- Feature file already contains a scenario with name 'Test'
- Inconsistent cell count within the table

General:
- A typo
![image](https://user-images.githubusercontent.com/48007045/121380721-cafa5400-c94d-11eb-9d15-df7ec2be0fd0.png)

**Comment by nemesv** [comment]
Fixed in version 1.8.0.
